homework-2_config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "client_id": "452925651630-egr1f18o96acjjvphpbbd1qlsevkho1d.apps.googleusercontent.com",
+  "client_secret": "1U3-Krii5x1oLPrwD5zgn-ry",
```

Comment:
You should not include this! These are the credentials to your google account. Make sure you remove this, generate a new set of keys and follow the instructions [here](https://gist.github.com/rickyc/aaedae5a21ddd21899658764513854f4#dotenv-gem).